test/inductor/test_torchinductor.py

```diff
@@ -6141,6 +6141,14 @@ def fn(x):
 
         self.common(fn, (torch.randn(8, 8),))
 
+    def test_uint(self):
+        def fn():
+            x = torch.tensor(5, device="cuda", dtype=torch.uint8)
```

Comment:
You can't specify "cuda" here because that will break when the test runs on cpu-only platforms.

Reply:
fixed in [remove changes in common.py; fix test unit](https://github.com/pytorch/pytorch/pull/101468/commits/8f6f6736e09b7bd08c23a99fbfbe6e68abdb9f23)